index.html

```diff
@@ -81,100 +81,14 @@ <h2>Multi-Post Stories </h2>
-                        <ul>
-                            <li><a href="" class="button">html</a></li>
-                            <li><a href="" class="button">bootstrap</a></li>
-                            <li><a href="" class="button disabled">Ruby</a></li>
-                        </ul>
-                    </div>
-                    <button type="button" class="link">See Project</button>
-                </div>
-            </div>
-            <div class="work work--4-4">
-                <div class="container">
-                    <h3 class="p1">Profesional Art Printing Data</h3>
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
-                        industry's standard</p>
-                    <div class="languages">
-                        <ul>
-                            <li><a href="" class="button">html</a></li>
-                            <li><a href="" class="button">bootstrap</a></li>
-                            <li><a href="" class="button disabled">Ruby</a></li>
-                        </ul>
-                    </div>
-                    <button type="button" class="link">See Project</button>
-                </div>
-            </div>
-            <div class="work work--4-5">
-                <div class="container">
-                    <h3 class="p1">Data Dashboard<br>Healthcare</h3>
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
-                        industry's standard</p>
-                    <div class="languages">
-                        <ul>
-                            <li><a href="" class="button">html</a></li>
-                            <li><a href="" class="button">bootstrap</a></li>
-                            <li><a href="" class="button disabled">Ruby</a></li>
-                        </ul>
-                    </div>
-                    <button type="button" class="link">See Project</button>
-                </div>
-            </div>
-            <div class="work work--4-6">
-                <div class="container">
-                    <h3 class="p1">Website Portfolio</h3>
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
-                        industry's standard</p>
-                    <div class="languages">
-                        <ul>
-                            <li><a href="" class="button">html</a></li>
-                            <li><a href="" class="button">bootstrap</a></li>
-                            <li><a href="" class="button disabled">Ruby</a></li>
-                        </ul>
-                    </div>
-                    <button type="button" class="link">See Project</button>
-                </div>
-            </div>
+            
         </div>
     </section>
     <section class="about-me" id="about">
```

Comment:
- [ ] Kindly add one validation checking if the email is written in all lowercase, If a user submits the contact form with the email field containing uppercase characters an error message should show up close to submit button when the user clicks the submit button.
- [ ] Kindly add a javascript code that checks if the email is all lowercase by comparing if the email and it's lowercase converted version are the same.